src/widgets/helper/ChannelView.cpp

```diff
@@ -1706,11 +1706,18 @@ void ChannelView::addContextMenuItems(
         menu->addSeparator();
     }
 
-    // Copy actions
+    // Selection actions
     if (!this->selection_.isEmpty())
     {
         menu->addAction("Copy selection",
                         [this] { crossPlatformCopy(this->getSelectedText()); });
+
+        menu->addAction("Search with Google", [this] {
+            auto query =
+                QString(QUrl::toPercentEncoding(this->getSelectedText()));
+            QDesktopServices::openUrl(
+                QUrl("https://google.com/search?q=" + query));
```

Comment:
I do not like this string being hardcoded here. Consider extracting the opening of the link into an own class, especially in regards to custom search query schemas potentially being added later on.